core/src/de/golfgl/gdxgamesvcs/player/IPlayerData.java

```diff
@@ -0,0 +1,44 @@
+package de.golfgl.gdxgamesvcs.player;
+
+/**
+ * Leaderboard entry data
+ *
+ * @author Kari Vatjus-Anttila
+ */
+public interface IPlayerData {
+
+    /**
+     * Player Id
```

Comment:
It's redundant information, just doc the return value

Reply:
Done